network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -21,16 +23,22 @@ <h1>{{ block.value }}</h1>
     </div>
 
     {% if page.cta != None %}
-    <div class="cta col-lg-{% if uses_menu == False %}4 offset-lg-1{% else %}12{% endif %}">
-        <div class="narrow-sticky-button-container hidden">
-          <a href="#cta-anchor" class="narrow-sticky-button btn">TAKE ACTION</a>
+    <div class="cta
+      {% if uses_menu == False and page.specific.narrowed_page_content == False %}
+        col-lg-4 offset-lg-1
+      {% else %}
+        col-lg-12
+      {% endif %}"
+    >
+      <div class="narrow-sticky-button-container hidden">
+        <a href="#cta-anchor" class="narrow-sticky-button btn">TAKE ACTION</a>
```

Comment:
Is this link supposed to exist on every campaign page?

Reply:
yeah, (from my understanding) `if page.cta != None` would mean there is a useful place to send people if they click that button.

Reply:
Any campaign page without a cta, there shouldn't be a button.